components/tabs/nz-tabs-nav.component.ts

```diff
@@ -286,6 +289,23 @@ export class NzTabsNavComponent implements AfterContentChecked, AfterContentInit
     }
   }
 
+  get navContainerScrollPaddingPix(): number {
+    if (this.platform.isBrowser) {
```

Comment:
miss `PlatformModule` in the module

Reply:
I'm confused about whether it's necessary to add `PlatformModule` in `ngModule`, i found that there is no `PlatformModule` in `NzAffix` and `NzCarouselModule`. Can you give me more reasons to do this? `Platform` is a `Service` which is `@Injectable({providedIn: 'root'})`。 Here is the link to [PlatformModule](https://github.com/angular/components/blob/master/src/cdk/platform/platform-module.ts) and [Platform](https://github.com/angular/components/blob/master/src/cdk/platform/platform.ts) in angular.

Reply:
it should be a bug in `NzAffixModule` and `NzCarouselModule` then, we have no idea what changes will happen in `PlatformModule` in the furture.